charts/fleet-telemetry/README.md

```diff
@@ -12,7 +12,7 @@ _See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation
 To install the chart with the release name `fleet-telemetry`:
 
 ```console
-helm install fleet-telemetry charts/fleet-telemetry -n fleet-telemetry
+helm install fleet-telemetry teslamotors/fleet-telemetry -n fleet-telemetry
```

Comment:
i think this change is not correct as we are talking about https://github.com/teslamotors/helm-charts/tree/main/charts/fleet-telemetry

Reply:
No it is actually correct because the first step in the README is adding this chart repo locally with the name `teslamotors`. The install command then references the repo name and the chart name. So the install command in the README does not work because the repo name referenced is `charts` but the repo was added with name `teslamotors`.

This PR is actually because the instructions did not work for me locally (Helm version 3.12.1), but it did work after changing the repo name from `charts` to `teslamotors` in the `helm repo add` command.

Reply:
good catch